engine/src/main/java/org/hibernate/search/engine/search/dsl/predicate/MatchPredicateFieldSetContext.java

```diff
@@ -69,13 +71,35 @@ default MatchPredicateFieldSetContext orRawField(String absoluteFieldPath) {
 
 	/**
 	 * Require at least one of the targeted fields to match the given value.
+	 * <p>
+	 * {@link DslConverter} option is {@code ENABLED} by default.
+	 * To change the option use {@link #matching(Object, DslConverter)} method instead.
```

Comment:
Suggestion: replace this:

```
	 * {@link DslConverter} option is {@code ENABLED} by default.
	 * To change the option use {@link #matching(Object, DslConverter)} method instead.
```

with this:

```
	 * This method will apply DSL converters to {@code value} before Hibernate Search attempts to interpret it as a field value.
	 * See {@link DslConverter#ENABLED}.
```

That way, we'll be able to move all the documentation related to DSL converters to... `DslConverter`.

By the way, before we merge, please remind me to move the javadoc currently located at `SearchPredicateFactoryContext.html#commonconcepts-parametertype` and `SearchPredicateFactoryContext.html#commonconcepts-rawfield` to `DslConverter`. I'll rephrase it as appropriate.